portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,12 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function showMoreInfo()
+{
```

Comment:
Minor nit: include the opening brace on the same line as the function.
ie, `async function showMoreInfo() {`